elixir/README.md

```diff
@@ -0,0 +1,193 @@
+# Welcome to Elixir-land!
+
+## Running Control Plane for local development
+
+You can use [Top-Level Docker Compose](../docker-compose.yml) and start any released Elixir
+app. `web` and `api` services are running application release that will be pretty much the same
+as the one we run in production, while `elixir` compose service runs raw Elixir code, without a release.
+
+It means that you can run any Elixir code including Mix tasks using `elixir` service but you can't do that
+in `web`/`api` so easily, because Elixir strips a lot of tooling during compilation.
```

Comment:
```suggestion
You can use the [Top-Level Docker Compose](../docker-compose.yml) to start any services locally. The `web` and `api` compose services are built application releases that are pretty much the same as the ones we run in production, while the `elixir` compose service runs raw Elixir code, without a built release.

This means you'll want to use the `elixir` compose service to run Mix tasks and any Elixir code on-the-fly, but you can't do that
in `web`/`api` so easily because Elixir strips out Mix and other tooling [when building an application release](https://hexdocs.pm/mix/Mix.Tasks.Release.html).
```